src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -28,6 +35,11 @@ export class AboutComponent implements OnInit, OnDestroy {
       this.versionHash = version[1];
       this.versionName = version.slice(2, version.length).join(' ');
     });
+    this.user = localStorage.getItem('dashboard_username');
+    this.browserName = detect().name;
+    this.browserVersion = detect().version;
+    this.browserOS = detect().os;
+    this.hostName = "Host";
```

Comment:
You can easily obtain the hostname from the getHealth() (health/ endpoint) JSON:

This is the ceph-mgr service endpoint:
`mgr_map.active_address = "172.17.0.2:6800/11514"`

And this is the dashboard specific one:
`mgr_map.services.dashboard = "https://ceph-dev:11000/"`

In any case, that's not a plain hostname but an service (protocol + hostname + port), so we should change the name of the property as well.

Reply:
@epuertat ack, maybe host address will be more accurate